src/main/java/reactor/netty/tcp/ProxyProvider.java

```diff
@@ -92,6 +96,12 @@ public static ProxyProvider findProxySupport(Bootstrap b) {
 		else {
 			this.nonProxyHosts = null;
 		}
+		if (Objects.isNull(builder.httpHeaders)) {
+			this.httpHeaders = () -> null;
```

Comment:
can we extract that `() -> null` as a constant

Reply:
ok. I try to rewrite it.